Tools/Scripts/generate-gpup-webgl

```diff
@@ -89,6 +89,9 @@ context_messages_template = (
     + """#if ENABLE(GPU_PROCESS) && ENABLE(WEBGL)
 
 messages -> RemoteGraphicsContextGL NotRefCounted Stream {{
+#if PLATFORM(COCOA)
+    void CreateAndBindExternalImage(GCGLenum target, WebCore::ExternalImageSource source) -> (uint64_t handle, WebCore::IntSize size) Synchronous
```

Comment:
possibly this will be changed by the learning experience, e.g this can go to the autogenerated section.

Reply:
This can't be generated automatically because `ExternalImageSource` contains `WTF::MachSendRight` and serializers can't be generated for `StreamConnectionEncoder`.